tests/unittest/_torch/multi_gpu_modeling/test_llama4.py

```diff
@@ -0,0 +1,51 @@
+# TODO: Enable this test when the models are checked into the llm repo!!
+from difflib import SequenceMatcher
+
+import pytest
+from utils.llm_data import llm_models_root
+
+from tensorrt_llm import SamplingParams
+from tensorrt_llm._torch import LLM
+from tensorrt_llm._torch.pyexecutor.config import PyTorchConfig
+
+
+@pytest.mark.parametrize(
+    "model_name",
+    ["Llama-4-Maverick-17B-128E-Instruct", "Llama-4-Scout-17B-16E-Instruct"],
+    ids=['maverick', 'scout'])
+@pytest.mark.parametrize("backend", ["TRTLLM", "FLASHINFER"],
```

Comment:
Does FLASHINFER actually work?

Reply:
Yeah, it works. I used it a lot for debugging. It's helpful because it does rope externally